examples/concept_learning_cv_evaluation.py

```diff
@@ -89,12 +89,14 @@ def dl_concept_learning(args):
               kwargs_classifier={"random_state": 1},
               max_runtime=args.max_runtime,
               verbose=0)
+    
     nces = NCES(knowledge_base_path=args.kb,
                 quality_func=F1(),
                 path_of_embeddings=args.path_of_nces_embeddings,
-                pretrained_model_name=["LSTM", "GRU", "SetTransformer"],
+                pretrained_model_name=["LSTM", "GRU", "SetTransformer"]
```

Comment:
```suggestion
                learner_names=["LSTM", "GRU", "SetTransformer"],
```
-> this should be learner names? Also why was the comma removed?

Reply:
after these changes this script runs, but still throws the following error:
`AttributeError: 'NoneType' object has no attribute 'shape'`

for line 63 in class BaseNCES:
`self.input_size = self.instance_embeddings.shape[1]`

Reply:
@LckyLke that happens because you need to pass the path of embeddings. 
There are a few other errors with this script which are not related with this PR. 
Ill open an issue for that.